src/NuGetGallery/Controllers/AccountsController.cs

```diff
@@ -338,7 +338,7 @@ public virtual ActionResult DeleteRequest(string accountName = null)
         }
 
         [HttpGet]
-        [UIAuthorize(Roles = "Admins")]
+        [AdminAction()]
```

Comment:
We should be able to get rid of the `()`:

```suggestion
        [AdminAction]
```